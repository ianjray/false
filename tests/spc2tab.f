{ convert SPC+ to TAB }
0 { state }
[^$1_≠]
[
 £0= [ $32= [ + 9 ]? ,
    ][ $32= [ %   ][ , %0 ]¿
    ]¿
]#%%
