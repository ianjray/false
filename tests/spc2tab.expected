|	1|
|	2|
|	3|
|4	abc|
|5d	ef|
|6gh	i|
||
|8|
|9	|
|A	|
|B	|
